### ISSUE TITLE
translation bugs on Sites page

### ISSUE THREAD

**Issue by axoplasm** [mercycorps/TolaActivity#601]
Search field label & submit button

http://127.0.0.1:8000/workflow/siteprofile_list/0/0/

![image](https://user-images.githubusercontent.com/169060/45379015-d315e100-b5b3-11e8-94c7-4103b7e6a9b7.png)

**Comment by axoplasm** [comment]
This bug is really weird. I've isolated [the code](https://github.com/mercycorps/TolaActivity/commit/9c06d697226eec514efb04d687b73cc939773488#diff-7979b84e6529ca79df37f27228111453) ensured the strings are [appropriately in the .po file](https://github.com/mercycorps/TolaActivity/commit/9c06d697226eec514efb04d687b73cc939773488#diff-11604557eb72d1bf768a6fdcea0bb4f4) & regenerated the .mo…but it still doesn't work. I blame Crispy forms. I am totally OK deferring this for a later release.

**Comment by esinn** [comment]
Agreed, let's defer. Those seem minor and we can come back to this.
